wallet/udb/addressmanager.go

```diff
@@ -1354,36 +1369,25 @@ func (m *Manager) IsLocked() bool {
 func (m *Manager) Lock() error {
 	// A watching-only address manager can't be locked.
 	if m.watchingOnly {
-		return managerError(ErrWatchingOnly, errWatchingOnly, nil)
+		return managerError(apperrors.ErrWatchingOnly, errWatchingOnly, nil)
 	}
 
 	m.mtx.Lock()
 	defer m.mtx.Unlock()
 
 	// Error on attempt to lock an already locked manager.
 	if m.locked {
-		return managerError(ErrLocked, errLocked, nil)
+		return managerError(apperrors.ErrLocked, errLocked, nil)
 	}
 
 	m.lock()
 	return nil
 }
 
-// lookupAccount loads account number stored in the manager for the given
-// account name
-//
-// This function MUST be called with the manager lock held for reads.
-func (m *Manager) lookupAccount(ns walletdb.ReadBucket, name string) (uint32, error) {
-	return fetchAccountByName(ns, name)
-}
-
 // LookupAccount loads account number stored in the manager for the given
 // account name
 func (m *Manager) LookupAccount(ns walletdb.ReadBucket, name string) (uint32, error) {
-	m.mtx.RLock()
-	defer m.mtx.RUnlock()
```

Comment:
Well this is an exported func and there's no need to comment about internal locking in the godoc comment.  I can mention the reason inside the function body though.